### PR TITLE
Update store definition and genesis asset schema - Closes #7572 and #7578

### LINKED ISSUE #7572: Update token module sub stores
### Description

Update the definition of new sub stores and update accordingly to the [PR](https://github.com/LiskHQ/lips/blob/08935fbff959dd14f1fc0c2b1b64890f904d5eae/proposals/lip-0051.md#token-module-store)

- Update constant `TOKEN_ID_LSK`

### Acceptance Criteria

- All stores mentioned in the LIP is implemented
- Additional logic to the stores are implemented
- All the changes are unit tested

### Additional Information
- `Supported Tokens Substore` is a new store
- Validation to the store is added

### LINKED ISSUE #7578: Update token module genesis state handling
### Description

Update genesis state handling (initGenesisState/finalizeGenesisState) and genesis block asset schema according to the [PR](https://github.com/LiskHQ/lips/blob/08935fbff959dd14f1fc0c2b1b64890f904d5eae/proposals/lip-0051.md#genesis-block-processing)

- Update commander `init` command to generate new block. The default genesis block is created using https://github.com/LiskHQ/lisk-sdk/blob/development/commander/src/utils/genesis_creation.ts
- Update genesis asset for dpos mainchain example

### Acceptance Criteria

- Updated genesis state handling logics are unit tested
- commander `init` command generates the genesis block which can be used
- Dpos mainchain runs and accept token transfer transaction

### Additional Information
- This change requires #7571 and #7572

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves  #7572 and resolves #7578

### How was it solved?

- Remove outdated token module stores
- Update genesis asset schema for token module

### How was it tested?

- Integration tests are updated
- Functional tests are working
- Test to create new application
- Test dpos-mainchain example is running
- tests for token methods/endpoint/cc commands are skipped for now

### CODE REVIEW

File: framework/test/unit/modules/token/stores/escrow.spec.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import { createStoreGetter } from '../../../../../src/testing/utils';
+
+describe('EscrowStore', () => {
+	const defaultSendingChainID = Buffer.from([2, 0, 0, 0]);
+	const defaultTokenID = Buffer.from([1, 0, 0, 0, 0, 0, 0, 0]);
+
+	let store: EscrowStore;
+	let context: StoreGetter;
+
+	beforeEach(async () => {
```
Comment (mosmartin) [inline framework/test/unit/modules/token/stores/escrow.spec.ts:28]: Async arrow function has no 'await' expression.

File: framework/src/modules/token/endpoint.ts [hunk 2]
```diff
@@ -113,12 +111,13 @@ export class TokenEndpoint extends BaseEndpoint {
 		};
 	}
 
+	// TODO: Update to use SupportedTokensStore
```
[the remark this answers is not in the document]
Comment (shuse2) [reply framework/src/modules/token/endpoint.ts]: Updated #7579 to include this changes

File: framework/src/modules/token/stores/supported_tokens.ts [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+		await this.del(context, ALL_SUPPORTED_TOKENS_KEY);
+		await this.set(context, chainID, supported);
+	}
+
+	private _isMainchainOrNative(tokenID: Buffer): boolean {
+		const [chainID] = splitTokenID(tokenID);
+		if (chainID.equals(this._ownChainID)) {
+			return true;
+		}
+		if (chainID[0] === this._ownChainID[0] && chainID.slice(1).equals(CHAIN_ID_LSK.slice(1))) {
```
Comment (mosmartin) [inline framework/src/modules/token/stores/supported_tokens.ts:132]: What do you think of replacing lines 132 - L135 with 
`return chainID[0] === this._ownChainID[0] && chainID.slice(1).equals(CHAIN_ID_LSK.slice(1));`

**Review by mosmartin — APPROVED** [review]
🚀

**Review by has5aan — APPROVED** [review]

**Pull request merged**